compute_tools/src/compute.rs

```diff
@@ -1367,6 +1368,38 @@ LIMIT 100",
         download_size
     }
 
+    pub fn install_extension(&self, ext_name: &str, db_name: &str) -> Result<String> {
+        let mut conf =
+            Config::from_str(self.connstr.as_str()).context("Failed to parse connection string")?;
```

Comment:
I think it should be good to just `unwrap` here. At this point, we've used the same connection string many times to connect.